pulseguardian/dbinit.py

```diff
@@ -50,34 +50,37 @@ def init_and_clear_db():
 
 def dummy_data():
     # Dummy test users
+    User.new_user(email='dummy1@dummy.com')
```

Comment:
Tiny thing: this should be dummy0@dummy.com just to match the pattern of the PulseUser usernames.